src/RoboFile/RoboFileBase.php

```diff
@@ -274,4 +274,49 @@ public function runCodeceptionSuite($suite, $test, $debug = false, $env = '')
 			->run()
 			->stopOnFail();
 	}
+
+	/**
+	 * Function for actual execution of a individual test of this extension
+	 *
+	 * @param   array  $opts  Array of configuration options:
+	 *                        - 'env': set a specific environment to get configuration from
+	 *                        - 'debug': executes codeception tasks with extended debug
+	 * 						  - 'suite': the suite of the test
+	 * 						  - 'test': example: 'install/InstallWeblinksCest.php/installWeblinks'
+	 * 						  - 'server': the container name on which the tests are run
+	 *
+	 * @return void
+	 *
+	 * @since   x.x.x
+	 */
+	public function runTestSingle(
```

Comment:
Sorry for insisting on this topic and maybe you had done it right the first time and I misunderstood.
I think this full function should be in the `ClientContainer` trait and doesn't have to be defined here because this class uses the `ClientContainer` trait